packages/emoji/src/utils/shortnameToUnicode.ts

```diff
@@ -1,13 +1,24 @@
-import {
-  shortnameToUnicode as emojiToolkitShortnameToUnicode,
-  toShort,
-} from 'emoji-toolkit';
+import { toShort } from 'emoji-toolkit';
 import data from 'emojibase-data/en/compact.json';
 
-const mapShortnameToUnicode: Record<string, string> = Object.fromEntries(
-  data.map((item) => [toShort(item.unicode), item.unicode])
-);
-
+const mapShortnameToUnicode: { key: string; value: string }[] = [];
+data.forEach((item) => {
+  mapShortnameToUnicode.push({
+    key: toShort(item.unicode),
+    value: item.unicode,
+  });
+  if (item.skins) {
+    item.skins.forEach((skin) => {
+      mapShortnameToUnicode.push({
+        key: toShort(skin.unicode),
+        value: skin.unicode,
+      });
+    });
+  }
+});
 export default function shortnameToUnicode(str: string): string {
-  return mapShortnameToUnicode[str] || emojiToolkitShortnameToUnicode(str);
+  return str.length
+    ? // Somehow encoding is different, checking with indexOf instead of === works
```

Comment:
Can you tell me in which case `===` does not work?
It would also be a better solution to use a map instead of iterating over the array.

Reply:
for example with `:thumbsup` it's not working.

I will change it to map, thx

Reply:
You wrote that there might be an issue with encoding, that why I asked. As far as I know this function is always called with the full shortname.
By map I meant the native `Map`.

Reply:
for example I got this:

`encodeURI(':thumbsup:️') --> ':thumbsup:%EF%B8%8F'`

Reply:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/10879009/211209885-4b5fcd9b-a0d4-453f-bc2e-9a20a1ed845c.png">


that's the issue in it's happening